build.sbt

```diff
@@ -4,7 +4,7 @@ import sbtrelease.ReleaseStateTransformations._
 
 import scala.sys.process._
 
-val scalatest = "org.scalatest" %% "scalatest" % "3.2.1"
+val scalatest = "org.scalatest" %% "scalatest" % "3.2.2"
```

Comment:
the fix I made that went into 3.2.1 turned out to break something else which was fixed in 3.2.2.  I was having trouble with it not telling me which test failed, just which sbt project, so I thought bumping it would be a good idea.